defi/src/api2/routes/tvl.ts

```diff
@@ -0,0 +1,304 @@
+import sluggify from "../../utils/sluggify";
+import { cachedCraftProtocolV2 } from "../utils/craftProtocolV2";
+import { cachedCraftParentProtocolV2 } from "../utils/craftParentProtocolV2";
+import * as sdk from '@defillama/sdk'
+import { get20MinDate } from "../../utils/shared";
+import { craftProtocolsResponseInternal as craftAllProtocolResponse } from "../../getProtocols";
+import type { IProtocol, IChain, } from "../../types";
+import { craftChainsResponse } from "../../getChains";
+import { getTokensInProtocolsInternal } from "../../getTokenInProtocols";
+import { successResponse, errorWrapper as ew } from "./utils";
+import { getContractNameInternal } from "../../getContractName";
+import { getRaisesInternal } from "../../getRaises";
+import { getHacksInternal } from "../../getHacks";
+import { readFromPGCache } from "../db";
+import { PG_CACHE_KEYS } from "../constants";
+import { getSimpleChainDatasetInternal } from "../../getSimpleChainDataset";
+import { getTotalChainDatasetInternal } from "../../getTotalChainDataset";
+import craftCsvDataset from "../../storeTvlUtils/craftCsvDataset";
+
+export default function setRoutes(webserver: HyperExpress.Server) {
+
+  webserver.get("/protocol/:name", ew(async (req: any, res: any) => getProtocolishData(req, res, { dataType: 'protocol', skipAggregatedTvl: false, useNewChainNames: false, })));
+  webserver.get("/treasury/:name", ew(async (req: any, res: any) => getProtocolishData(req, res, { dataType: 'treasury' })));
+  webserver.get("/entity/:name", ew(async (req: any, res: any) => getProtocolishData(req, res, { dataType: 'entities' })));
+  webserver.get("/updatedProtocol/:name", (async (req, res) => getProtocolishData(req, res, { dataType: 'protocol', useHourlyData: false, skipAggregatedTvl: req.query_parameters.includeAggregatedTvl !== 'true' })));
+  webserver.get("/hourly/:name", (async (req, res) => getProtocolishData(req, res, { dataType: 'protocol', useHourlyData: true, skipAggregatedTvl: false })));
+
+  webserver.get("/protocols", ew(async (req: any, res: any) => getAllProtocolishData(req, res, { dataType: 'protocol', includeTokenBreakdowns: false, useNewChainNames: false, })));
+  webserver.get("/treasuries", ew(async (req: any, res: any) => getAllProtocolishData(req, res, { dataType: 'treasury' })));
+  webserver.get("/entities", ew(async (req: any, res: any) => getAllProtocolishData(req, res, { dataType: 'entities' })));
+
+  webserver.get('/chains', ew(async (req: any, res: any) => getChainsData(req, res, { isV2: false })))
+  webserver.get('/v2/chains', ew(async (req: any, res: any) => getChainsData(req, res, { isV2: true })))
+
+  webserver.get("/tvl/:name", ew(getProtocolTvl));
+  webserver.get("/tokenProtocols/:symbol", ew(getTokenInProtocols));
+  webserver.get("/config/smol/:protocol", ew(getSmolConfig));
+  webserver.get("/config/:chain/:contract", ew(getContractName));
```

Comment:
yeah, thought about it, will leave it up but not use. Btw, noticed that these responses are cached for 30 minutes, can update it to day or so since they are not likely to change often?